MetaMask/metamask-extension
# Remove confirmation for terminated snaps

## Explanation
Removes confirmation prompts for snaps that have been terminated.

## More information


Fixes #14494

## Manual testing steps

1. Go to https://metamask.github.io/test-snaps/0.1.3/
2. Connect the confirm snap
3. Click "send inputs to hello snap"
4. Wait 60 seconds
5. See that prompt is removed
